src/backend/distributed/commands/citus_add_local_table_to_metadata.c

```diff
@@ -46,6 +46,8 @@
 #include "utils/syscache.h"
 
 
+bool AddAllLocalTablesToMetadata = true;
```

Comment:
maybe add a comment on how this is used, especially since it's not used in this file